src/libclipper/include/clipper/logging.hpp

```diff
@@ -0,0 +1,172 @@
+#ifndef CLIPPER_LOGGING_HPP
+#define CLIPPER_LOGGING_HPP
+
+#include <string>
+#include <sstream>
+
+#include <spdlog/spdlog.h>
+
+namespace clipper {
+
+static constexpr uint LOGGING_FORMAT_LENGTH = 18;
+static constexpr uint MAX_TAG_LENGTH = 10;
```

Comment:
More generally, a maximum tag length is required to preserve the alignment of log messages. Should I increase the length from 10?